domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -87,4 +88,10 @@ object InteractionObjectTestBuilder {
   fun createInt(value: Int): InteractionObject {
     return InteractionObject.newBuilder().setReal(value.toDouble()).build()
   }
+
+  fun createRatio(value: List<Int>): InteractionObject {
```

Comment:
There are changes in these non xml files too. Thats because your branch is still 3 commits behind current develop.

If you can just merge with latest develop this should work and make these changes not visible.

Reply:
Done, It is still yet to be fix